spring-cloud-dataflow-rest-resource/src/main/java/org/springframework/cloud/dataflow/rest/job/support/JobUtils.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.rest.job.support;
+
+import org.springframework.batch.core.BatchStatus;
+import org.springframework.batch.core.JobExecution;
+import org.springframework.util.Assert;
+
+/**
+ * Provides a common utility methods for job-related tasks.
+ *
+ * @author Gunnar Hillert
+ * @since 1.0
+ */
+public final class JobUtils {
+
+	/** Prevent instantiation. */
+	private JobUtils() {
+		throw new AssertionError();
```

Comment:
Not required by it is recommended by Josh Bloch :-) 

http://www.informit.com/articles/article.aspx?p=1216151&seqNum=4